src/block.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env mocha */
+/* globals apiClients */
+'use strict'
+
+const expect = require('chai').expect
+
+module.exports = (common) => {
+  describe.only('.block', () => {
+    const blorbKey = 'QmPv52ekjS75L4JmHpXVeuJ5uX2ecSfSZo88NSyxwA3rAQ'
+    const blorb = Buffer('blorb')
+
+    it('returns an error when putting an array of files', () => {
+      return apiClients.a.block.put([blorb, blorb], (err) => {
```

Comment:
This seems that it is just prepared for js-ipfs-api and leverages the global.

See https://github.com/ipfs/interface-ipfs-core/blob/master/src/files.js#L26-L35 to learn how to get the `ipfs` object.